spec/features/publishing_a_cma_case_spec.rb

```diff
@@ -174,6 +180,8 @@
 
       expect(page).to have_no_content("You are about to publish a minor edit")
       expect(page).to have_no_content("You are about to publish a major edit")
+
+      expect(page.find_button('Publish')["data-message"]).to eq("Publishing will email subscribers to CMA Cases. Continue?")
```

Comment:
What do you think about DRYing up these scenarios by having something like:

`let(:publish_alert_message) { page.find_button('Publish')["data-message"] }` somewhere at the top of the file?